partselector.py

```diff
@@ -600,6 +600,24 @@ def update_subcategories(self, *_):
         # search now that categories might have changed
         self.search(None)
 
+    def get_price(self, quantity, prices):
```

Comment:
could I suggest that we round to say three decimal points? like if price > 0.001 then price = round(ceiling(price), 3)?

Two arguments for dropping precision:
- This provides the per-board cost and if it's like $0.001 dollars total then it might as well be zero.

- If the cost of these passives matters for volume we'd need some way to let the user enter total board count per year or something to give them a total part cost. I usually manage my bom in a separate spreadsheet where I can calculate 1k pricing and a total burdened cost etc with margin etc.

Reply:
Oh and it would make it easier to read that column, the small price parts are something I ignore so having a column that mostly lines up like 0.001 values would make it easier to skim over them.